ledger/src/blockstore_meta.rs

```diff
@@ -214,6 +241,25 @@ impl SlotMeta {
         Some(self.consumed) == self.last_index.map(|ix| ix + 1)
     }
 
+    pub fn is_connected(&self) -> bool {
```

Comment:
I think we should check both ConnectedFlags::CONNECTED and ConnectedFlags::PARENT_CONNECTED true.

Reply:
Given that `set_connected()` asserts that `is_parent_connected()` and we don't have a way to unset connected, this is overkill IMO.